website/src/models/group.js

```diff
@@ -168,6 +170,81 @@ schema.methods.isMember = function isGroupMember (user) {
   }
 };
 
+schema.methods.startQuest = async function startQuest (user) {
+  if (this.type !== 'party') throw new BadRequest('Must be a party to use this method');
+  if (!this.quest.key) throw new BadRequest('Party does not have a pending quest');
+  if (this.quest.active) throw new BadRequest('Quest is already active');
+
```

Comment:
@TheHollidayInn the fact is that we already have checks in the controllers for the user input, the errors here can only happen if a developer calls the questStart method with in the wrong situazion or with the wrong parameters, it's our fault